docs/instrumentation-list.yaml

```diff
@@ -436,239 +570,331 @@ http:
   instrumentations:
   - name: http-url-connection
     srcPath: instrumentation/http-url-connection
+    scope:
+      name: io.opentelemetry.http-url-connection
     target_versions:
-      javaagent: []
+      javaagent:
+      - Java 8+
 hystrix:
   instrumentations:
   - name: hystrix-1.4
     srcPath: instrumentation/hystrix-1.4
+    scope:
+      name: io.opentelemetry.hystrix-1.4
     target_versions:
       javaagent:
       - com.netflix.hystrix:hystrix-core:[1.4.0,)
 influxdb:
   instrumentations:
   - name: influxdb-2.4
     srcPath: instrumentation/influxdb-2.4
+    scope:
+      name: io.opentelemetry.influxdb-2.4
     target_versions:
       javaagent:
       - org.influxdb:influxdb-java:[2.4,)
 internal:
   instrumentations:
   - name: internal-application-logger
     srcPath: instrumentation/internal/internal-application-logger
+    scope:
+      name: io.opentelemetry.internal-application-logger
     target_versions:
       javaagent:
       - org.springframework.boot:spring-boot:[1.2.0,)
       - org.slf4j:slf4j-api:[1.4.0,)
   - name: internal-class-loader
     srcPath: instrumentation/internal/internal-class-loader
+    scope:
+      name: io.opentelemetry.internal-class-loader
     target_versions:
       javaagent: []
   - name: internal-lambda-java9
     srcPath: instrumentation/internal/internal-lambda-java9
+    scope:
+      name: io.opentelemetry.internal-lambda-java9
     target_versions: {}
   - name: internal-reflection
     srcPath: instrumentation/internal/internal-reflection
+    scope:
+      name: io.opentelemetry.internal-reflection
     target_versions:
       javaagent: []
   - name: internal-lambda
     srcPath: instrumentation/internal/internal-lambda
+    scope:
+      name: io.opentelemetry.internal-lambda
     target_versions:
       javaagent: []
   - name: internal-eclipse-osgi-3.6
     srcPath: instrumentation/internal/internal-eclipse-osgi-3.6
+    scope:
+      name: io.opentelemetry.internal-eclipse-osgi-3.6
     target_versions:
       javaagent: []
   - name: internal-url-class-loader
     srcPath: instrumentation/internal/internal-url-class-loader
+    scope:
+      name: io.opentelemetry.internal-url-class-loader
     target_versions:
       javaagent: []
 java:
   instrumentations:
   - name: java-util-logging
     srcPath: instrumentation/java-util-logging
+    scope:
+      name: io.opentelemetry.java-util-logging
     target_versions:
       javaagent: []
   - name: java-http-server
     srcPath: instrumentation/java-http-server
+    scope:
+      name: io.opentelemetry.java-http-server
     target_versions:
-      javaagent: []
+      javaagent:
+      - Java 8+
       library: []
   - name: java-http-client
     srcPath: instrumentation/java-http-client
+    scope:
+      name: io.opentelemetry.java-http-client
     target_versions:
-      javaagent: []
+      javaagent:
+      - Java 8+
```

Comment:
I think ideally this one would say Java 11+

Reply:
thanks for pointing this out, i've now updated the logic to look for the minimum java version and update the coreJDK output accordingly. 

We can also now expose the minimum java version supported for each instrumentation, which i think is helpful